README.md

```diff
@@ -86,8 +86,9 @@ In the popup which will open set "Calendar API" to "enabled" (click the
 switch on its row on the right).  
 Once you have done this open the link which says "Google API Console" in a new tab: you
 will be taken to another page. In this page search for "Google Calendar API" and
-open it. Now click "Enable" at the top of the window and close this page. Back where you toggled the switch ("Resources" -> "Advanced Google services") make sure "Calendar API" is still on and press "Okay".
-That's it for this step.
+open it. Now click "Enable" at the top of the window and close this page.  
+Back where you toggled the switch ("Resources" -> "Advanced Google services") make sure "Calendar API" is still on and press "Okay".  
+That's it for this step.  
```

Comment:
OK, you fixed the trailing spaces on the line that ends with ` "Okay".`. The trailing spaces didn't need adding to this next line though (ending with `this step.`), they weren't there before. From looking at it rendered in the browser that seems to make no difference due to the new paragraph anyway, so don't worry too much about that until @GioBonvi responds about the PR itself.